Website/src/rendering/src/components/HeroSection.tsx

```diff
@@ -1,50 +1,40 @@
-import { ComponentProps } from 'lib/component-props';
-import { Text, Field, LinkField, Link, ImageField, Image } from '@sitecore-jss/sitecore-jss-nextjs';
+import { Text, Field, RichText, ImageField } from '@sitecore-jss/sitecore-jss-nextjs';
+import { ComponentWithChildrenProps } from 'lib/component-props';
+import { Placeholder } from '@sitecore-jss/sitecore-jss-nextjs';
 
-export type HeroProps = ComponentProps & {
+export type HeroProps = ComponentWithChildrenProps & {
   fields: {
     Hero: ImageField;
-    Logo: ImageField;
     Slogan: Field<string>;
-    Expo: Field<string>;
+    Eyebrow: Field<string>;
     Title: Field<string>;
-    Subtitle: Field<string>;
-    When: Field<string>;
-    Link: LinkField;
+    Body: Field<string>;
   };
 };
 
-const HeroSection = ({ fields }: HeroProps): JSX.Element => {
+const HeroSection = (props: HeroProps): JSX.Element => {
   return (
     <section
       className="section__hero banner"
-      style={{ backgroundImage: `url("${fields.Hero?.value?.src}")` }}
+      style={{ backgroundImage: `url("${props.fields.Hero?.value?.src}")` }}
     >
       <div className="section__hero__container container">
         <div className="section__hero__container__content">
           <div className="section__hero__container__content__text">
-            <div className="logo">
-              <Image field={fields.Logo} alt="Logo" loading="lazy" />
-            </div>
             <p className="slogan">
-              <Text field={fields.Slogan} />
+              <Text field={props.fields.Slogan} />
             </p>
             <h1 className="expo">
-              <Text field={fields.Expo} />
+              <Text field={props.fields.Eyebrow} />
             </h1>
             <h3 className="title">
-              <Text field={fields.Title} />
+              <Text field={props.fields.Title} />
             </h3>
             <p className="subtitle">
-              <Text field={fields.Subtitle} />
+              <RichText field={props.fields.Body} />
             </p>
-            <h3 className="expo">
-              <Text field={fields.When} />
-            </h3>
-          </div>
-          <div className="btn__area">
-            <Link field={fields.Link} className="btn--main btn--main--round btn--main--big" />
           </div>
+          <Placeholder name="jss-summit-hero-cta" rendering={props.rendering} />
```

Comment:
The name of this placeholder is not following the other placeholders naming convention (`jss-[component-name]-content`. For this component, it should be `jss-hero-section-content`. Please also change the placeholder name in the content tree.

Reply:
Updated